src/main.zig

```diff
@@ -268,6 +268,7 @@ const RLPEncodablePerson = struct {
     age: u8,
 
     pub fn encodeToRLP(self: RLPEncodablePerson, list: *ArrayList(u8)) !void {
+        _ = self;
```

Comment:
why not renaming `self` as `_` here?

Reply:
This is the default way the compiler fmt autofixes unused variables (since that isn't allowed).
So.. just left what the format decided.